### PR TITLE
naming & tests for grr vector function

### PR DESCRIPTION
**Pull request by exaexa**
small cleanups

### CODE REVIEW

**Comment by cylon-x** [comment]
:green_circle: &nbsp;Commit 69ef4a54 is formatted properly.

**Review by stelmo — APPROVED** [review]

File: src/base/types/CoreModel.jl [hunk 1]
```diff
@@ -103,11 +103,12 @@ reaction_stoichiometry(m::CoreModel, ridx)::Dict{String,Float64} =
     Dict(m.mets[k] => v for (k, v) in zip(findnz(m.S[:, ridx])...))
 
 """
-    grrs(a::CoreModel)::Vector{Maybe{GeneAssociation}}
+    reaction_gene_association_vec(model::CoreModel)::Vector{Maybe{GeneAssociation}}
 
-Get the gene associations in a [`CoreModel`](@ref).
+Retrieve a vector of all gene associations in a [`CoreModel`](@ref), in the
+same order as `genes(model)`.
```
Comment (htpusa) [inline src/base/types/CoreModel.jl:109]: Same order as reactions?

Comment (exaexa) [reply src/base/types/CoreModel.jl:109]: aah good catch

**Comment by cylon-x** [comment]
:green_circle: &nbsp;Commit cfc1b5d1 is formatted properly.

**Pull request merged**